src/Redux/missions/MissionsSlice.js

```diff
@@ -26,6 +27,7 @@ const data = (missions) => {
       mission_id: mission.mission_id,
       mission_name: mission.mission_name,
       description: mission.description,
+      reserve: false,
```

Comment:
Kindly consider deleting this key, I could notice that you already add the key in the actions so if you delete this line nothing in the functionalities change

Reply:
Thanks @AndreaM2429 for noticing it! I have remove it! 🙌